src/components/screens/IndexScreen/Hero.js

```diff
@@ -389,7 +389,7 @@ export default function Hero({ startOpen, ...props }) {
   return (
     <Wrapper {...props}>
       <Pitch>
-        <Title>Build bulletproof UI components faster</Title>
+        <Title>Build component driven UIs faster</Title>
```

Comment:
```suggestion
        <Title>Build component-driven UIs faster</Title>
```

Reply:
To be more consistent

Reply:
I was on the fence about the hyphen. 

We aren't consistent on componentdriven.org; we use "Component Driven User Interfaces" (no hyphen) there.

If anything we'd capitalize "Component Driven" since it's a proper noun.